assets/css/unsupported-theme.scss

```diff
@@ -0,0 +1,31 @@
+/**
+ * unsupported-theme.scss
+ * Styles applied to themes that don't formally support WooCommerce to normalize elements.
+ */
+
+.single-product {
+
+	// Hide duplicate product title.
+	h1.product_title {
+		display: none;
+	}
+
+	// Normalize the button and input to the same height and font size.
```

Comment:
Let's try and roll these styles (this and below) into our default CSS if possible. Having dedicated CSS for unsupported themes seems weird to me?